servers/src/main/java/tachyon/worker/block/BlockWorker.java

```diff
@@ -163,9 +161,6 @@ public BlockWorker() throws IOException {
     // Setup user metadata mapping
     // TODO: Have a top level register that gets the worker id.
     long workerId = mBlockMasterSync.getWorkerId();
-    String tachyonHome = mTachyonConf.get(Constants.TACHYON_HOME);
-    String ufsAddress =
-        mTachyonConf.get(Constants.UNDERFS_ADDRESS);
```

Comment:
Thanks for getting rid of these lines, but in the future, could you try to keep the changes isolated to addressing the ticket?